### DISCUSSION TITLE
Is there a chance to increase the OS of Oracle Database container images from OL7 to OL8

### DISCUSSION THREAD

**Discussion by Khan-Tzar** [oracle/docker-images#2564] [Q&A]
Currently Oracle Database container images are set to run on Oracle Linux 7 slim. Is there a chance to increase to Oracle Linux 8 slim?
Thank you a lot

**Comment by kristianzlatkov** [comment]
I was wondering the same thing since we don't use OL7 and migrated to OL8. Is this possible?
Thank you in advance.
